wai-aria/role/invalid-roles.html

```diff
@@ -11,26 +11,109 @@
 </head>
```

Comment:
To account for adding the unescaped unicode chars.
```suggestion
  <meta charset="utf-8">
</head>
```